src/css/imports/mixins.less

```diff
@@ -256,6 +256,17 @@
 
     }
 
+    /* Override default touch flag styles */
+    .touch-flag-skin-styles() {
+        &.jw-flag-touch {
```

Comment:
This flag only increases the size of the controlbar on larger breakpoints. I think we're fine for this release to merge. Improvements to `jw-flag-touch` scaling with various breakpoints will come with upcoming mobile-ui work.